backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -44,7 +47,27 @@ object VeloxBackendSettings extends BackendSettings {
   override def supportExpandExec(): Boolean = true
   override def needProjectExpandOutput: Boolean = true
   override def supportSortExec(): Boolean = true
-  override def supportWindowExec(): Boolean = true
+  override def supportWindowExec(windowFunctions: Seq[NamedExpression]): Boolean = {
+    var allSupported = true
+    breakable {
+      windowFunctions.foreach(
+        func => {
+          val aliasExpr = func.asInstanceOf[Alias]
+          val wExpression = aliasExpr.child.asInstanceOf[WindowExpression]
+          wExpression match {
+            case rowNumber: RowNumber => allSupported = allSupported & true
+            case aggExpression: AggregateExpression => allSupported = allSupported & true
+            case rank: Rank => allSupported = allSupported & true
+            case lead: Lead => allSupported = allSupported & true
+            case lag: Lag => allSupported = allSupported & true
```

Comment:
Currently, for velox backend, Lead/Lag are not supported. We should set `allSupported` to false here, right?

Reply:
Yes